### ISSUE TITLE
BeagleBone Blue doesn't save config

### ISSUE THREAD

**Issue by BluePackStack** [beagleboard/beaglebone-blue#12]
I am not sure whether I am allowed to ask this question here, but any help is pretty much appreciated. I tried to change the wifi config. 
`connmanctl> tether wifi disable` ...
But no effect . What can I do?

- currently running/booting  Debian 9.2 2017-10-10 4GB SD IoT  from sd card
- already pluged in 12Vdc adapter to improve power-same result
- tried to `shutdown -r now` 
- and replaced the image on sd card with this one: Debian 9.1 2017-09-21 4GB SD IoT image

Hope someone can help. :(

**Comment by RobertCNelson** [comment]
@BluePackStack all the out of box control of wl18xx for the default image is located in:

```
/etc/default/bb-wl18xx
```
Regards,

**Comment by BluePackStack** [comment]
After restarting. Changes in `/etc/default/bb-wl18xx` are finally done. Very nice!

 Is it true that connman is very buggy on beaglebone? Is there an alternative you can recommend?

**Comment by RobertCNelson** [comment]
It's not buggy, just highly intergrated for the out of box experience... I've setup /etc/default/bb-wl18xx so you can disable parts of it..

**Comment by BluePackStack** [comment]
> It's not buggy, just highly intergrated for the out of box experience... I've setup /etc/default/bb-wl18xx so you can disable parts of it..<

I cleared my early response and went to bed then, since I recognized, that the board wouldn't response to ssh anymore after editing the settings. It seems I can sometimes manipulate softAp SSID and password through different ways, but at the cost of losing control over the board every time. Always have to rewrite SD/Memory to get into system again.

Using: bone-debian-9.2-iot-armhf-2017-10-10-4


Anyway I am very happy that you help me! I will add further information about what exactly crashes my board.
